### PR TITLE
#2215 - Add new CIP codes

### PR DESCRIPTION
**Pull request by dheepak-aot**
# Add new CIP Codes

 - [x] Default the `Field of study code` to 20 for any CIP code which is not mapped
image

 - [x] Add new CIP codes which are highlighted in brown in the spreadsheet `General > Analysis > User Story Specifications > Federal Study Code (#1222) > Fed Study Code Mapping.xslx` (Also mentioned in user story)
image

image

image
 - [x] Modified the regex based validation for `CIP code` from values to only pattern based regex validation. This is because we can now enter a CIP code which is not mapped in the system.
image

image

### CODE REVIEW

File: sources/packages/forms/src/form-definitions/educationprogram.json [hunk 1]
```diff
@@ -858,7 +860,7 @@
                     "redrawOn": "",
                     "clearOnHide": true,
                     "customDefaultValue": "",
-                    "calculateValue": "value = data.fieldOfStudyCodes[data.sabcCodes[data.cipCode.substring(0, 2)] + data.levelOfStudyCodes[data.credentialType]];",
+                    "calculateValue": "const sabcCode = data.sabcCodes[data.cipCode.substring(0, 2)];\r\nconst levelOfStudyCode =  data.levelOfStudyCodes[data.credentialType];\r\nvalue = (sabcCode && levelOfStudyCode) ? data.fieldOfStudyCodes[sabcCode + levelOfStudyCode] : 20;",
```
Comment (andrepestana-aot) [inline sources/packages/forms/src/form-definitions/educationprogram.json:863]: 👍

**Review by andrepestana-aot — APPROVED** [review]
Looks good!

**Review by lewischen-aot — APPROVED** [review]
LGTM well done 👍

File: sources/packages/backend/package-lock.json [hunk 2]
```diff
@@ -12389,4 +12389,4 @@
       }
     }
   }
-}
\ No newline at end of file
+}
```
Comment (ann-aot) [inline sources/packages/backend/package-lock.json:12392]: do we need this change?

**Review by ann-aot — APPROVED** [review]
LGTM 👍 Nice work @dheepak-aot . just a minor comment

**Review by guru-aot — APPROVED** [review]
LGTM, nice work @dheepak-aot

File: sources/packages/backend/package-lock.json (hunk 2 above)
Comment (dheepak-aot) [reply sources/packages/backend/package-lock.json:12392]: I really  didn't want to add it. But whenever I ran the code(npm install && npm run start) taking upstream from main, I saw it getting updated, Hence I added.

**Pull request merged**